docs/testing/cypress.mdx

```diff
@@ -0,0 +1,193 @@
+---
+title: Testing with Cypress
+description: Use Cypress to write end-to-end tests with Clerk.
+---
+
+# Testing with Cypress
+
+> [!WARNING]
+> Our `@clerk/testing` package supports only end-to-end testing with Cypress. Unit tests are not supported.
```

Comment:
Does it not also support E2E testing with playwright? If so, let's write:

```suggestion
> Our `@clerk/testing` package only supports end-to-end testing. Unit tests are not supported.
```

Reply:
Hmm, actually yes, we only support E2E tests, but Playwright has no unit tests either way. So, I added this note only here because it affects mainly Cypress, but we can make it more generic as you suggest, no problem.